sql/core/src/test/scala/org/apache/spark/sql/SQLQueryTestSuite.scala

```diff
@@ -317,110 +317,112 @@ class SQLQueryTestSuite extends QueryTest with SharedSparkSession {
     // Create a local SparkSession to have stronger isolation between different test cases.
     // This does not isolate catalog changes.
     val localSparkSession = spark.newSession()
-    loadTestData(localSparkSession)
+    localSparkSession.withActive {
```

Comment:
This is test and the SparkContext will be closed once the test suite is done. Maybe just set `localSparkSession` as active session is good enough?

Reply:
Actually if we make sure that all SparkSession dataset methods are covered by withActive, then this should not be an issue.

Reply:
+1